views/builder/index.jade

```diff
@@ -18,10 +18,12 @@ block main
 
 					label
 						input(type="checkbox", name="minified", value="1")
+						span
 						| Download minified source code
```

Comment:
@arian no, the span is just a "holder" for the square showing the selected/not selected state.